nilearn/_utils/data_gen.py

```diff
@@ -45,10 +45,10 @@ def generate_mni_space_img(n_scans=1, res=30, random_state=0, mask_dilation=2):
     Returns
     -------
     inverse_img : Niimg-like object
-        Image transformed back to mni space.
+        Image transformed back to MNI space.
```

Comment:
```suggestion
        Image transformed back to :term:`MNI space.
```

Reply:
Technically I ignored private functions and functions in private modules